pkgs/by-name/pd/pdfium-binaries/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p bash nixVersions.latest curl coreutils jq common-updater-scripts
+
+latestTag=$(curl -s ${GITHUB_TOKEN:+" -u \":$GITHUB_TOKEN\""} https://api.github.com/repos/bblanchon/pdfium-binaries/releases/latest | jq -r ".tag_name")
+latestVersion=$(echo $latestTag | sed 's/^chromium\///')
```

Comment:
Nit: You can strip the `chromium` prefix easily in jq itself :P
`jq -r '.tag_name | ltrimstr("chromium")'`